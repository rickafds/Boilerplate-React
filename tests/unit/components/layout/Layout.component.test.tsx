import React from 'react';
import { shallow } from 'enzyme';

import Layout from '@/components/layout/Layout.component';

describe('<Layout />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<Layout />);
    expect(wrapper).toMatchSnapshot();
  });
});
