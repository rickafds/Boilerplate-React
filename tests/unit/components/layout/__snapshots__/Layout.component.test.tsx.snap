// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should match snapchat 1`] = `
<Layout
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <Content />
</Layout>
`;
